picky-krb/src/constants.rs

```diff
@@ -33,24 +39,29 @@ pub mod types {
     pub const ENC_TGS_REP_PART_TYPE: u8 = 26;
     pub const ENC_AP_REP_PART_TYPE: u8 = 27;
 
+    /// [Address Types](https://datatracker.ietf.org/doc/html/rfc4120#section-7.5.3)
     pub const IP_V4_ADDR_TYPE: u8 = 2;
-    pub const DIRECTIONAL_ADDR_TYPE: u8 = 2;
-    pub const CHAOS_NET_ADDR_TYPE: u8 = 2;
-    pub const XNS_ADDR_TYPE: u8 = 2;
-    pub const ISO_ADDR_TYPE: u8 = 2;
-    pub const DECNET_PHASE_IV_ADDR_TYPE: u8 = 2;
-    pub const APPLETALK_DDP_ADDR_TYPE: u8 = 2;
-    pub const NET_BIOS_ADDR_TYPE: u8 = 2;
-    pub const IP_V6_ADDR_TYPE: u8 = 2;
+    pub const DIRECTIONAL_ADDR_TYPE: u8 = 3;
+    pub const CHAOS_NET_ADDR_TYPE: u8 = 5;
+    pub const XNS_ADDR_TYPE: u8 = 6;
+    pub const ISO_ADDR_TYPE: u8 = 7;
+    pub const DECNET_PHASE_IV_ADDR_TYPE: u8 = 12;
+    pub const APPLETALK_DDP_ADDR_TYPE: u8 = 16;
+    pub const NET_BIOS_ADDR_TYPE: u8 = 20;
+    pub const IP_V6_ADDR_TYPE: u8 = 24;
 }
 
 pub mod key_usages {
+    /// [GSS API Key Usages](https://datatracker.ietf.org/doc/html/rfc4121#section-2)
     pub const ACCEPTOR_SEAL: i32 = 22;
     pub const ACCEPTOR_SIGN: i32 = 23;
     pub const INITIATOR_SEAL: i32 = 24;
     pub const INITIATOR_SIGN: i32 = 25;
+    /// [Key Usage Numbers](https://datatracker.ietf.org/doc/html/rfc4120#section-6.2)
+    pub const KRB_PRIV_ENC_PART: i32 = 13;
```

Comment:
I don’t see this constant in the link above (https://datatracker.ietf.org/doc/html/rfc4120#section-6.2)

Reply:
Yeah. It's from a different section of the RFC. I fixed it.

Reply:
Yeah, It's from a different section of the RFC. I fixed it.